.azure-pipelines/linux/basic-tests.yml

```diff
@@ -0,0 +1,35 @@
+#
+# Azure Pipelines configuration for running tests and other checks on Linux/macOS.
+#
+
+steps:
+  - script: yarn prettier-ci
+    displayName: 'Prettier'
+
+  - script: node ./scripts/tasks/flow-ci
+    displayName: 'Flow CI'
+
+  - script: |
+      export JEST_JUNIT_OUTPUT=.azure-pipelines/test-results/test-basic-linux.xml
+      yarn test --maxWorkers=2
+    displayName: 'Run tests'
+
+  - script: ./scripts/circleci/check_license.sh
```

Comment:
They may balk at this (and suggest this gets done in a separate PR), but what about renaming `scripts/circleci` to  `scripts/ci`? One more thing in a theme of generalizing and becoming less tied to any one CI service.

Reply:
Yeah, I was going to suggest it in the PR description but leave it out for now.